lib/generators/react_on_rails/react_no_redux_generator.rb

```diff
@@ -10,22 +10,22 @@ class ReactNoReduxGenerator < Rails::Generators::Base
       Rails::Generators.hide_namespace(namespace)
       source_root(File.expand_path("templates", __dir__))
 
+      BASE_PATH = "base/base"
```

Comment:
why is this duplicated in several files?

Reply:
It was already duplicated in those files.
It might be good to extract it to the `Generators` module, but I am not sure yet.

Reply:
Changed back